src/session.js

```diff
@@ -108,13 +108,48 @@ class Session {
   }
 
   /**
+  * Function used to get an API for a backend object.
+  * @param {Object} args Arguments used to create object API.
+  * @param {Number} args.handle Handle of the backend object.
+  * @param {String} args.id ID of the backend object.
+  * @param {String} args.type QIX type of the backend object. Can for example
+  *                           be "Doc" or "GenericVariable".
+  * @param {String} args.customType Custom type of the backend object, if defined in qInfo.
+  * @returns {*} Returns the generated and possibly augmented API.
+  */
+  getObjectApi(args) {
+    const { handle, id, type, customType } = args;
+    let api = this.apis.getApi(handle);
+    if (api) {
+      return api;
+    }
+    api = this.definition
+      .generate(type)
+      .create(this, handle, id, this.protocol.delta, customType);
+    this.apis.add(handle, api);
+    return api;
+  }
+
+  handleObjectCreationResponse(response) {
```

Comment:
This also handles `getObject`, `openDoc` etc. so perhaps `handleObjectReferenceResponse` or something?